cmd/drone-server/server.go

```diff
@@ -101,12 +101,13 @@ var flags = []cli.Flag{
 	cli.DurationFlag{
 		EnvVar: "DRONE_SESSION_EXPIRES",
 		Name:   "session-expires",
-		Usage:  "Set the session expiration time default 72h",
+		Usage:  "set the session expiration time default 72h",
 		Value:  time.Hour * 72,
 	},
 	cli.StringSliceFlag{
 		EnvVar: "DRONE_ESCALATE",
 		Name:   "escalate",
+		Usage:  "images run in privileged mode",
```

Comment:
`images and services to run in privileged mode`?

Reply:
services are images, no? ;)

Reply:
Yeah true, thought about plugins and services. :D